ee/server/calendar/google_calendar.go

```diff
@@ -30,7 +30,7 @@ import (
 //    to create multiple events in the same calendar. This is useful for load testing. For example: john+test@example.com becomes john@example.com
 
 const (
-	eventTitle  = "💻🚫Downtime"
+	eventTitle  = "💻 🚫 Scheduled maintenance"
 	startHour   = 9
```

Comment:
There's two other occurences of `💻🚫Downtime` (in tools). Let's update those too.

Reply:
Thanks! Updated.